deisctl/cmd/platform.go

```diff
@@ -11,12 +11,18 @@ import (
 
 // InstallPlatform loads all components' definitions from local unit files.
 // After InstallPlatform, all components will be available for StartPlatform.
-func InstallPlatform(b backend.Backend, checkKeys func() error) error {
+func InstallPlatform(b backend.Backend, checkKeys func() error, stateless bool) error {
 
 	if err := checkKeys(); err != nil {
 		return err
 	}
 
+	if stateless {
+		fmt.Println("Warning: With a stateless control plane, `deis logs` will be unavailable.")
+		fmt.Println("Additionally, components will need to be configured to use external persistent stores.")
+		fmt.Println("See the official Deis documentation for details on running a stateless control plane.")
```

Comment:
> it already works that way

I know you had mentioned changing the controller to just `Wants` instead of `Requires` `deis-store-volume`, but I think @Blystad said it'll come up just fine either way. Does this mean we can just skip the mock `deis-store-volume` entirely?